public-ui/src/app/views/quick-search/quick-search.component.ts

```diff
@@ -189,5 +196,18 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
     localStorage.setItem('searchText', this.prevSearchText);
     this.router.navigateByUrl('/ehr/' + r.domain.toLowerCase());
   }
+  public viewPhysicalMeasurements(searchString: string) {
+    this.router.navigateByUrl('/physical-measurements/' + searchString.toLowerCase());
+  }
+  public matchPhysicalMeasurements(searchString: string) {
+      if (searchString) {
```

Comment:
I think this is missing some cases. Probably cleaner would be:

```
if (!this.pmConceptGroups) {
  return 0;
}
if (!searchString) {
 return this.pmConceptGropus.length
}
return filter...
```

Reply:
Addressed. Thanks.